AutoDarkModeApp/Pages/PageSettings.xaml.cs

```diff
@@ -163,7 +163,7 @@ private async Task GetAutostartInfo(bool noToggle = false, bool toggleVisibility
                         ButtonAutostartValidate.IsEnabled = true;
                         CheckBoxLogonTask.IsEnabled = true;
                         if (!noToggle) ToggleAutostart.IsOn = true;
-                        TextBlockAutostartMode.Text = "Registry";
```

Comment:
This also must not be translated. If people submit screenshots of this it will be difficult to discern what mode is active.